specification/2.0/README.md

```diff
@@ -956,6 +956,8 @@ The following example shows an image pointing to an external PNG image file and
     ]
 }
 ```
+> **Implementation Note:** When image data is provided by `uri` and `mimeType` is defined, clients should prefer JSON-defined MIME Type over one provided by transport layer.
```

Comment:
Okay. I only asked because I haven't seen any implementations that take `mimeType` into account when `uri` is specified.